WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m

```diff
@@ -249,12 +253,20 @@ - (BOOL)fb_pressButton:(NSString *)buttonName error:(NSError **)error
              withDescriptionFormat:@"The button '%@' is unknown. Only the following button names are supported: %@", buttonName, supportedButtonNames]
             buildError:error];
   }
-  [[XCUIRemote sharedRemote] pressButton:remoteButton];
+
+  (duration < 0)
+    // https://developer.apple.com/documentation/xctest/xcuiremote/1627476-pressbutton
+    ? [[XCUIRemote sharedRemote] pressButton:remoteButton]
```

Comment:
I would rather use vanilla `if` there. Some optimizing compilers might skip such contructions completely as potential dead code, since the result of equation is not assigned to anything

Reply:
yea, looks good. I wondered which was better... (first I did `nil`)